expertise/models/specter2_scincl/scincl.py

```diff
@@ -118,6 +119,7 @@ def set_archives_dataset(self, archives_dataset):
                     self.pub_note_id_to_abstract[publication['id']] = publication['content'].get('abstract').strip() if publication['content'].get('abstract').strip() else "."
                     pub_mdate = publication.get('mdate', int(time.time()))
                     pub_cache_key = publication['id'] + "_" + str(pub_mdate)
+                    pub_weight = publication.get('content', {}).get('weight', 1) ## Mention that default weights are 1
```

Comment:
I think all publications should have pub_weight at this point. The get_pub_weight method should add 1 by default so that we don't add it all the way here. It's clearer when all the logic related to how weights are gotten is in one place.